components/BankPayouts/data.ts

```diff
@@ -0,0 +1,15 @@
+export interface MyProps {
+    id?: string
+    type?: string
+    label: string
+    title: string
+    desc: string
+}
+
+export const periphery: MyProps[] = [{
+    id: 'email',
+    type: 'email',
+    label: 'email',
+    title: 'Get Notified When Bank Payout is Live',
+    desc: 'Bank Payout to multiple countries in USD and other local currencies is coming soon. Reach out to developers@lazerpay.finance if you have any questions regarding this'
```

Comment:
To add formatting to the link, please use html code. Then render the value on the text as 
```js
{
   desc: `Bank Payout to multiple countries in USD and other local currencies is coming soon. Reach out to <a href="mailto:developers@lazerpay.finance">developers@lazerpay.finance</a> if you have any questions regarding this`
}
```
```jsx
<p className='paragraph-2' dangerouslySetInnerHTML={{__html: desc}} />
```